src/testsHelpers.ts

```diff
@@ -0,0 +1,33 @@
+// @ts-ignore
+export async function asyncResultsToValue(jestFned) {
+  // @ts-ignore
+  jestFned.mock.results = await Promise.all(
+    (jestFned.mock.results || []).map(
+      async (result: { value: any; type: any }) => {
+        if (result.value instanceof Promise) {
+          return {
+            type: result.type,
+            wasPromise: true,
+            value: await result.value
+          };
+        }
+
+        return result;
+      }
+    )
+  );
+
+  return jestFned;
+}
+
+export function createJestFnSerialReturn(toReturn: any[]) {
```

Comment:
Please comment such functions where it's hard to understand what/why is this happening